Vagrantfile

```diff
@@ -145,6 +154,17 @@ Vagrant.configure("2") do |config|
 		end
 	end
 
+	config.vm.provider "parallels" do |prl|
+		# Use memory and CPU settings from yaml config file.
+		# The Virtual Machine customizations use `virtualbox` as
+		# the key for backward compatibility but apply to the
+		# Parallels provider too.
```

Comment:
```suggestion
```

Reply:
Thanks @peterwilsoncc. I thought your suggestion was a little confusing as it read like we're still using `virtualbox` so I just added the following in both of the places we use the customisations:
# The `virtualbox` value has been deprecated to use `virtualmachine`.